packages/captp/test/test-gc.js

```diff
@@ -21,6 +21,12 @@ test('test loopback gc', async t => {
 
   await isolated(t, makeFar);
   await gcAndFinalize();
-  t.is(getFarStats().sendCount.CTP_DROP, 3);
-  t.is(getNearStats().recvCount.CTP_DROP, 3);
+
+  // Check the GC stats.
+  const { sendCount: nearSend, recvCount: nearRecv } = getNearStats();
```

Comment:
here's a way to test this without more bindings:
```js
t.like(getNearStats(), {
  sendCount: {
    CTP_DROP: 4,
  },
  recvCount: {
    DROPPED: 2,
  },
};
```

Reply:
Thanks!